inst/extdata/installPackages.R

```diff
@@ -34,18 +60,59 @@ installPackages <- function(pkgs, lib, repos = getOption("repos")) {
   }
 }
 
-lib <- Sys.getenv("R_LIBS")
-if (nchar(lib) == 0) {
-  stop("Could not get a value for R_LIBS environment variable")
+nodename <- Sys.info()["nodename"]
+
+# if this is our Very Special development machine
+if (nodename == "IGSWZTWWWSASHA") {
+  lib <- .libPaths()[1]
+} else {
+  # it's the general public
+  lib <- Sys.getenv("R_LIBS")
+  
+  if (nchar(lib) == 0) {
+    stop("Could not get a value for R_LIBS environment variable")
+  }
 }
 
-# all packages except devtools and its prerequisites are held back to older
-# versions
-installPackages(pkgs, lib, "https://mran.microsoft.com/snapshot/2016-03-31")
+pkgs <-
+  grep(
+    "gsplot", repgenImports(lib.loc = lib),
+    value = TRUE, fixed = TRUE, invert = TRUE
+  )
+
+# all packages are held back to older, MRAN versions
+installPackages(c(pkgs, "devtools"), lib, "https://mran.microsoft.com/snapshot/2016-03-31")
+
+if (nodename == "IGSWZTWWWSASHA") {
```

Comment:
yes, please remove infrastructure specific logic (as well as the root cert specific functionality)